### ISSUE TITLE
Credit card section of checkout is missing aria-describedby attributes and has improperly escaped translation strings.

### ISSUE THREAD

**Issue by Shelob9** [easydigitaldownloads/easy-digital-downloads#5332]
PR incoming to fix.

**Comment by cklosowski** [comment]
As our primary gateways are using card fields more securely from the gateways themselves, I'm going to close this unless we have a need to approach this again.
